templates/angular/igx-ts/grid/grid-summaries/files/src/app/__path__/__name__.component.ts

```diff
@@ -13,7 +13,7 @@ import { employeesData } from './localData';
 export class GridSummariesComponent implements OnInit {
```

Comment:
The classname should be templated. -> $(ClassName)Component

if you add two template with any name the build will fail.